dotcom-rendering/src/web/server/newsletters-page-model/defaultData.ts

```diff
@@ -0,0 +1,265 @@
+			subscribe:
+				'https://support.theguardian.com/subscribe?INTCMP=header_support_subscribe&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_support_subscribe%22%7D',
+			support:
+				'https://support.theguardian.com?INTCMP=amp_header_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22amp_header_support%22%7D',
+			supporter:
+				'https://support.theguardian.com/subscribe?INTCMP=header_supporter_cta&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_supporter_cta%22%7D',
+		},
+		ampFooter: {
+			contribute:
+				'https://support.theguardian.com/contribute?INTCMP=amp_footer_support_contribute&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_support_contribute%22%7D',
+			subscribe:
+				'https://support.theguardian.com/subscribe?INTCMP=amp_footer_support_subscribe&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_support_subscribe%22%7D',
+			support:
+				'https://support.theguardian.com?INTCMP=footer_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22footer_support%22%7D',
+			supporter:
+				'https://support.theguardian.com/subscribe?INTCMP=amp_footer_supporter_cta&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_supporter_cta%22%7D',
+		},
+	},
+};
+
+export const STATIC_CONFIG: ConfigType = {
+	isPaidContent: false,
+	pageId: 'unknown-page',
+	contentType: 'static',
+	ampIframeUrl:
+		'https://assets.guim.co.uk/data/vendor/b242a49b1588bb36bdaacefe001ca77a/amp-iframe.html',
+	ajaxUrl: 'https://api.nextgen.guardianapps.co.uk',
+	shortUrlId: '/p/d8ex5',
+	switches: {},
+	keywordIds: '',
+	sharedAdTargeting: {},
+	dcrSentryDsn: 'https://1937ab71c8804b2b8438178dfdd6468f@sentry.io/1377847',
+	discussionApiUrl: 'https://discussion.theguardian.com/discussion-api',
+	sentryPublicApiKey: '344003a8d11c41d8800fbad8383fdc50',
+	commercialBundleUrl:
+		'https://assets.guim.co.uk/javascripts/bc58c17d75809551440f/graun.commercial.dcr.js',
+	discussionApiClientHeader: 'nextgen',
+	shouldHideReaderRevenue: false,
+	sentryHost: 'app.getsentry.com/35463',
+	idApiUrl: 'https://idapi.theguardian.com',
+	showRelatedContent: true,
+	adUnit: '/59666047/theguardian.com/environment/article/ng',
+	stage: 'DEV',
+	isSensitive: false,
+	revisionNumber: 'DEV',
+	section: 'environment',
+	brazeApiKey: '7f28c639-8bda-48ff-a3f6-24345abfc07c',
```

Comment:
This field name is wrong, it does not reference an API key (which should be secret) instead the string is the app name, so the field should be called `brazeAppIdentifier`.

Reply:
That's true, but unfortunately this is the name set by frontend (see https://github.com/guardian/frontend/blob/main/common/app/views/support/JavaScriptPage.scala ) and already used in the existing data models throughout dotcom-rendering.  It's also the name that's exposed on the frontend as `window.guardian.config.page.brazeApiKey`, so presumably there are some client-side scripts that rely on the that being the name used.

I agree this should be changed, but it would be confusing if the model names for this page didn't match what the rest of the project and frontend is doing. It would be best to address this on another PR so it can be applied consistently.

I'll add some comments about this for future reference.